functions/New-DbaLogin.ps1

```diff
@@ -103,6 +103,11 @@
 
         Copies logins [Login1] and [Login2] from instance sql1 to instance sql2, but enforces password and expiration policies for the new logins. New logins will also have a default database set to [tempdb] and will be created in a disabled state.
 
+    .EXAMPLE
```

Comment:
You need to revert this file back to the version in the current development branch. You included this update in #4163